index.js

```diff
@@ -149,6 +149,26 @@ function findTag(gitPath, sha) {
   return tags.length ? tags[0] : false;
 }
 
+function findLastTag(gitPath, sha, noOfAdditionalCommits) {
+  noOfAdditionalCommits = noOfAdditionalCommits || 0;
+  var tag = findTag(gitPath, sha);
```

Comment:
This is going to read and scan `.git/packed-refs` for each commit from `HEAD` to whatever has a tag, or all commits if there are no tags.

If we're going to always run `findLastTag`, we may want to cache `packed-refs`.  As I recall `git-repo-info` winds up getting called a lot in certain builds.  Is that right @rwjblue?